automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookInstallSpec.scala

```diff
@@ -50,8 +50,8 @@ class NotebookInstallSpec extends ClusterFixtureSpec with NotebookTestUtils {
       s"should be able to pip install packages using ${kernel.string}" in { clusterFixture =>
         withWebDriver { implicit driver =>
           withNewNotebook(clusterFixture.cluster, kernel) { notebookPage =>
-            // install tensorflow
```

Comment:
since we are now installing tensorflow by default, we have to install a package that isn't already present for this test to pass